gulpfile.js

```diff
@@ -206,7 +206,7 @@ gulp.task('build', function() {
  */
```

Comment:
There's a large number of CSS Lint failures now. We might want to exclude `public/viewer` from the CSS linting